scripts/gbp_format_changelog

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Simple changelog entry formatter
+#
+# It simply uses the built in formatter and linewraps the text
+#
+# Use git-dch --customizations=/usr/share/doc/git-buildpackage/examples/wrap_cl.py
+# or set it via gbp.conf
+
+import textwrap
+import re
+import gbp.dch
+
+
+FILTER_CLOUD_INIT_UPSTREAM_COMMITERS = (
```

Comment:
We're going to apply this to projects that are not cloud-init (e.g. cloud-utils), but I think it's fine to use cloud-init as a "flag project" for tableflip. Maybe worth adding a 1-line comment on this.

I don't think there's a straightforward way to differentiate per-project from the gbp dch customization file.

Reply:
> I don't think there's a straightforward way to differentiate per-project from the gbp dch customization file.

Agreed, I tried seeing if there was project-specific or repo-specific information in the commit_info or options dict that gbp passed to the call but I can't see anything distinguishing.

Reply:
Maybe we could pass the package name down to `gbp dch` / `gbp_format_changelog` using an environment variable, but that's not a nice solution and not worth it at the moment IMHO.

Reply:
I think we can look to adapt or grow this if needed. Given that gpb cusomtization conf files would need to live in debian/* anyway, adding a supplemental formatting file to a project's debian/* dir seems to make sense for the moment. Given that our primary consumers of this repo here are likely cloud-utils, ubuntu-advantage-tools, cloud-init and curtin we can make sure those projects work as advertized. Even if no defaults for gbp_format_changelog are changed. The only project-specific missing configuration is that changelog entries will not redact the upstream author names or specific JIRA ticket references. I don't believe the other projects are using JIRA tickets in their commit footers anyway, so no impact there.